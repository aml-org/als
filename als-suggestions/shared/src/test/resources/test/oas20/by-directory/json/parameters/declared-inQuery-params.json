{
  "swagger": "2.0",
  "paths": {
    "/users?userId=id&param=p": {
      "get": {
        "summary": "Gets a user by ID.",
        "parameters":
        [
          {
            "in": "query",
            "name": "*"
          }
        ]
      }
    }
  }
}
