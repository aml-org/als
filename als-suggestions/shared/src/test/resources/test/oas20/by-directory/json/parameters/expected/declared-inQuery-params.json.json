[
  {
    "label": "param",
    "kind": [],
    "detail": "unknown",
    "documentation": "param",
    "deprecated": [],
    "preselect": [],
    "sortText": "10param",
    "filterText": "\"param\"",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 10,
          "character": 20
        },
        "end": {
          "line": 10,
          "character": 22
        }
      },
      "newText": "\"param\""
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "userId",
    "kind": [],
    "detail": "unknown",
    "documentation": "userId",
    "deprecated": [],
    "preselect": [],
    "sortText": "10userId",
    "filterText": "\"userId\"",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 10,
          "character": 20
        },
        "end": {
          "line": 10,
          "character": 22
        }
      },
      "newText": "\"userId\""
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]